docker/Dockerfile.exp

```diff
@@ -4,13 +4,17 @@ FROM esmvalgroup/esmvalcore:latest
 
 COPY ./environment.yml /src/ESMValTool/environment.yml
 WORKDIR /src/ESMValTool
-RUN conda update -n base -y conda && conda install -y -c conda-forge mamba && mamba env create --name esmvaltool --file environment.yml && conda clean --all -y
```

Comment:
Please consider that every RUN command will result in a [new container](https://docs.docker.com/engine/reference/builder/#run), so it is important to end every RUN command with cleanup instructions. Forgetting to clean up will result in needlessly large containers.

Reply:
cheers @bouweandela - I've now added the main mamba operations in one RUN and added the cleanup after, yet to see if it runs well (it should, but you never know)

Reply:
passe :partying_face: